GitUI/UserControls/RevisionGrid/CopyContextMenuItem.cs

```diff
@@ -64,29 +70,38 @@ void OnDropDownOpening(object sender, EventArgs e)
 
                     foreach (var name in tagNames)
                     {
-                        AddItem(name, _ => name, Images.Tag);
+                        AddImmutableItem(name, Images.Tag);
                     }
 
                     DropDownItems.Add(new ToolStripSeparator());
                 }
 
                 // Add other items
-                AddItem($"{Strings.CommitHash}     ({revision.ObjectId.ToShortString()}...)", r => r.Guid, Images.CommitId, Keys.Control | Keys.C);
-                AddItem($"{Strings.Message}     ({revision.Subject.ShortenTo(30)})", r => r.Body ?? r.Subject, Images.Message);
-                AddItem($"{Strings.Author}     ({revision.Author})", r => r.Author, Images.Author);
+                AddItem(Strings.CommitHash,  (GitRevision r) => r.Guid, Images.CommitId, Keys.Control | Keys.C);
+                AddItem(Strings.Message,     r => r.Body ?? r.Subject, Images.Message);
+                AddItem(Strings.Author,      r => r.Author, Images.Author);
+                AddItem(Strings.AuthorDate,  r => r.AuthorDate.ToString(), Images.Date);
```

Comment:
This is a change, Author/CommitDate were only separated if they differed before.
I preferred the previous way, but this is a simplification, so approving.

Reply:
Another change is that the preview of what is to be copied has been removed in the case that there's only a single item selected.

Reply:
Yes. I prefered to have a unified display between the two use cases (multiple commits vs single commit selected)

Reply:
I also slightly prefer the previous presentation. It made it clear what you were going to copy.

Reply:
It is hard to remove existing features, those that loose functionality will be louder than that do not care. I have some preference to the existing implementation too, but can live with the change. The preview is good if you do not know what the field contains, Similar for date, this simplifies, you do not have to copy both (or even visually compare) if Author-date==Commit-date.

Some more approvals needed for merge - I do not know if changes are needed for that

The core change to copy many commits is good, all commands should be like that.